lib/graphs/abstract_digraph.nit

```diff
@@ -0,0 +1,583 @@
+# Implementations for representing directed graphs, also called digraphs.
+#
+# Currently, there are two classes
+module abstract_digraph
+
+# Interface for digraphs
+abstract class AbstractDigraph[V]
```

Comment:
I agree. What about naming it ``Digraph`` and the concrete implementation ``HashMapDigraph``?